src/platform/errors/interactiveCellResultError.ts

```diff
@@ -3,7 +3,16 @@
 
 import { NotebookCell } from 'vscode';
 
-// Small helper error to use in our class
+/**
+ * Error thrown when running cells in the interactive window.
+ *
+ * Cause:
+ * Kernel.executeCell() returns an error when running a cell in the interactive window.
+ *
+ * Handled by:
+ * Running cell in IW shows the error. If no running cell, then a notification is shown to the user.
+ *
+ */
```

Comment:
Was thinking if we wanted a handling category, I could refactor all of the errors to implement some interface with an enum that indicates the handling category (similar to the category already on the BaseError)

Reply:
It would be like:
```
enum HandlingType {
   'CellOutput'
   'Dialog'
   'Notification'
   'Swallowed'
}
```

Reply:
I wouldn't want to constrict an error type to be handled in a specific way - perhaps they all have a single way of being handled now, but that might not hold true if a new context is introduced. 
I think the handled by clause here is fine, if more nuance is introduced, the comment could simply be updated